src/sample/classes/Student.fxml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<?import javafx.scene.control.Button?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.control.TextField?>
+<?import javafx.scene.layout.AnchorPane?>
+
+<AnchorPane fx:id="anchorid" maxHeight="-Infinity" maxWidth="-Infinity" minHeight="-Infinity" minWidth="-Infinity" prefHeight="287.0" prefWidth="430.0" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1" fx:controller="sample.classes.StudentController">
+   <children>
+      <Button fx:id="button" layoutX="253.0" layoutY="128.0" mnemonicParsing="false" onAction="#handleButtonAction" prefHeight="30.0" prefWidth="101.0" text="Browse" />
+      <TextField fx:id="textfield" layoutX="24.0" layoutY="131.0" />
+      <Label fx:id="label" layoutX="24.0" layoutY="81.0" prefHeight="15.0" prefWidth="318.0" text="Choose directory to store your course list PDF FIle." />
+      <Button fx:id="confirm" layoutX="174.0" layoutY="216.0" mnemonicParsing="false" onAction="#confirmbutton" prefHeight="26.0" prefWidth="59.0" text="Confirm" />
+   </children>
```

Comment:
This file should be deleted, because we rename the file ady, so you can delete

Reply:
Okay, deleted

Reply:
Push it after you delete